pygsp2/plotting.py

```diff
@@ -415,7 +415,8 @@ def normalize(x):
         Set intercept value in G.plotting["normalize_intercept"]
         with value in [0, 1], default is .25.
         """
-        ptp = x.ptp()
+        #ptp = x.ptp()
+        ptp = np.ptp(x)
```

Comment:
Why this change not related to the name change?

Reply:
The .ptp() method was deprecated in favor of np.ptp()

Reply:
OK. However, it is not related to the PyGSP to PyGSP2 change. Let's keep it anyway.